docs/rest-api/concepts/scan-guidance.md

```diff
@@ -87,7 +99,7 @@ If your processing requires downloading the contents of an individual file, you
 
 ### Scanning permissions hierarchies
 
-By default, the delta query response will include sharing information for all items in the query that changed even if they inherit their permissions from their parent and did not have direct sharing changes themselves.  This typically then results in a follow up call to get the permission details for every item rather than just the ones whose sharing information changed.  You can optimize your understanding of how permission changes happen by adding the "Prefer: hierarchicalsharing" header to your delta query request.
+By default, the delta query response will include sharing information for items even if they inherit their permissions from their parent and did not have direct sharing changes themselves.  Note that the delta query response only includes items with direct changes to their content or metadata.  This typically then results in a follow up call to get the permission details for every item rather than just the ones whose sharing information changed.  You can optimize your understanding of how permission changes happen by adding the "Prefer: hierarchicalsharing" header to your delta query request.
```

Comment:
Regarding `only includes items with direct changes to their content or metadata` - that's not entirely accurate is it? We'll return parents as well, and there are scenarios where none of their metadata has changed.